### PR TITLE
Add details on storing the API key

### LINKED ISSUE #6: Add instructions about dandi-cli credentials management
https://github.com/dandi/dandi-cli/pull/267 is introducing to dandi-cli support for a fall back (if system wide credentials store is not available) to specify an alternative from the keyring's default - encrypted keyring. It would require password .

We should have a subsection in https://github.com/dandi/handbook/blob/master/docs/10_using_dandi.md which describes that and also provides a way for a "fully automated" setups, e.g. via env variable or establishing a plain text credential store (which would not require password).

### PR DESCRIPTION
**Pull request by jwodder**
Closes #6.

### CODE REVIEW

**Comment by satra** [comment]
should we add a line how to set the environment variable and move the rest of the details to a different section? this section seems really intense for anyone who does not know what a keyring is.

can we think of the simplest path for troubleshooting (across platforms).

**Comment by yarikoptic** [comment]
environment variable is not needed AFAIK if keyring is used -- makes it easy for a user etc.  But indeed let's move the details into a dedicated section "Access credentials" or alike and just reference it here.  @jwodder please adjust this PR and let's have it merged.

**Comment by satra** [comment]
now that we have migrated, could this be checked again, and any conflicts resolved?

**Comment by jwodder** [comment]
@yarikoptic @satra PR updated.

File: docs/10_using_dandi.md [hunk 1]
```diff
@@ -89,9 +89,13 @@ button.
         - Create a Python environment (e.g., Miniconda, virtualenv)
         - Install the DANDI CLI into your Python environment
 
-            `pip install "dandi>=0.14"`
+                pip install "dandi>=0.14"
```
Comment (satra) [inline docs/10_using_dandi.md:92]: btw, the .et file in dandi-cli hasn't been updated to reflect bad versions.

**Pull request merged**